### DISCUSSION TITLE
The pen and highlighter does not draw on pdf or even the blank document

### DISCUSSION THREAD

**Discussion by rajesh22** [xournalpp/xournalpp#3112] [General]
My Xournal installation in Ubuntu does not draw by pen or by highlighter. I can only write the text. I have tried many colors for pen. I see same issue even when I want to draw o/ annotate on plan or lined background. I can insert the text by selecting T. 
Seeing the manual checked the .xournal/configfile and discard_corepointer=false is ensured. 
Can some one help why I am not able to draw line or figure using xournalpp.

**Comment by rolandlo** [comment]
Did you install Xournal or Xournal++ (xournalpp)? Xournal is a different software and we can't help. In case you have Xournal++ installed: Which version? Try version [1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly). Check your settings in the input system panel under menu `Edit > Preferences`.

**Reply by rajesh22** [reply]
Dear Rolando, 
Thanks much for quick response. I have installed xournalpp. 
1. It was not installing using apt install xournal. So I had to use the snap install xournalpp. I am not able to see the menu at all in this hence can not see Preference values. 
2. I tried to install the link given by you. I installed "xournalpp-1.1.0.dev-nightly.20210429-Ubuntu-bionic-x86_64.deb".
It was not coming up. So used command line to execute the xournalpp.
I got error "/usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.22' not found"
3. Then I searched and installed 
sudo apt-get install gcc-4.9
sudo apt-get upgrade libstdc++6

Verified the GLIBCXX_3.4.22   is available. 

4. I ran the xournalpp again and getting error.

xournalpp: symbol lookup error: xournalpp: undefined symbol: _ZNK9portaudio6System14DeviceIteratorneERKS1_

DO I have build the .deb file locally. 

Thanks much for help and support in advance.

**Reply by rolandlo** [reply]
Well, `apt install xournal` would install `Xournal` and not `Xournal++`. So you got the Snap version and also tried the nightly relase for Ubuntu bionic (18.04)? Are you really on Ubuntu 18.04? Or what system do you use?

**Reply by rajesh22** [reply]
Minor correction - I had used >>> apt install xournalpp 
with snap also I used "xournalpp"
I am using Ubuntu 16.04, updated and upgraded.

Thanks much,

**Comment by rolandlo** [comment]
Ubuntu 16.04 is no longer supported. You can try to compile the latest master from source as described in [the wiki](https://github.com/xournalpp/xournalpp/blob/master/readme/LinuxBuild.md). Otherwise try to install an older version (1.0.20 or older) from the [releases page](https://github.com/xournalpp/xournalpp/releases).
